docs/cluster-registration.md

```diff
@@ -68,10 +68,10 @@ Repeat this step to unjoin any additional clusters.
 Joining additional clusters to a namespaced control plane requires
 providing additional arguments to `kubefedctl join`:
 
-- `--kubefed-namespace=<namespace>` to ensure the cluster has been registered
+- `--kubefed-namespace=<namespace>` to ensure the cluster has been joined
```

Comment:
Note the help text for `kubefedctl join` - `Join registers a Kubernetes cluster with a KubeFed control plane.`.  It would read funny if it were 'join joins a cluster with the control plane'.  We even debated renaming `join` to `register-cluster` but decided to retain the name of the command for now.